README.md

```diff
@@ -1,75 +1,43 @@
-# @cloudfour/eslint-plugin
+# @cloudfour/eslint-config
```

Comment:
Fun bit of history here — This repo is called `eslint-config` and originally, the node package was called `@cloudfour/eslint-config` with a version number of `0.X`.

One of Caleb's first tasks was bumping the version number to `1.0.0`.

Then he converted it to a super-plugin, and changed the package name to `@cloudfour/eslint-plugin` and bumped the version number to `2.0.0`.

Since then, we've released 20+ versions, bumping whenever a dependency released a major version.

Now, I'm converting it back to the config, and reclaiming the `@cloudfour/eslint-config` package name — which is still at 1.0.0, making this release 2.0.0!

Reply:
After further consideration, I've restored the changelog and set the version number to 24. No sense throwing out our history just because the package name changed again.